src/components/home/index.tsx

```diff
@@ -3,22 +3,21 @@ import React from "react"
 import Banner from "./Banner"
 import Features from "./Features"
 import Graph from "./Graph"
-import LegacyGateway from "./TheProblem"
+import LegacyGateway from "./Benefits"
 import MoreFeatures from "./MoreFeatures"
 import Partners from "./Partners"
 import Discover from "../shared/Discover"
 import Configuration from "./Configuration"
 import Playground from "./Playground"
+import Customer from "./Customers"
 
 const HomePage = (): JSX.Element => {
   return (
     <div className="">
       <Banner />
       <Configuration />
-      <Partners />
```

Comment:
We don't delete partners. We have moved it below.

Reply:
We have shifted this component under Benefits component